src/tests/zzaccommodation.test.js

```diff
@@ -0,0 +1,109 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import GenerateToken from '../helpers/token.helper';
+import db from '../database/models';
+import EncryptPassword from '../helpers/Encryptor';
+import app from '../app';
+
+chai.use(chaiHttp);
+chai.should();
+
+const token = GenerateToken({ email: 'nsengimanavedadom@gmail.com', isVerified: 'true', id: 1 });
+
+describe('Like or unlike accommodation Tests', () => {
+  before(async () => {
+    await db.user.destroy({ where: {}, force: true });
+    await db.notification.destroy({ where: {}, force: true });
+    await db.locations.destroy({ where: {}, force: true });
+    await db.user.create({
+      id: 198,
+      firstName: 'Veda',
+      lastName: 'Dominique',
+      email: 'nsengimanavedadom@gmail.com',
+      gender: 'male',
+      country: 'Rwanda',
+      birthdate: '12-04-1994',
+      password: EncryptPassword('vedadom'),
+      phoneNumber: '0785571790',
+      isVerified: true,
+      token,
+      emailNotification: true,
+      appNotification: true
+    });
+
+    await db.locations.create({
+      id: 87,
+      city: 'Nairobi'
+    });
+
+    await db.accomodation.create({
+      id: 142,
+      name: 'Marriot',
+      description: 'very good',
+      locationId: 87,
+      category: 'family',
+      owner: 'jordan',
+      numberOfRooms: 6,
+      availableRooms: 6
+    });
+    await db.accomodation.create({
+      id: 143,
+      name: 'Marriot',
+      description: 'very good',
+      locationId: 87,
+      category: 'family',
+      owner: 'jordan',
+      numberOfRooms: 6,
+      availableRooms: 0
+    });
+  });
+  it('should book an accommodation facility', (done) => {
+    chai.request(app).post('/api/v1/accommodations/booking')
+      .set('token', `Bearer ${token}`)
+      .send({
+        userId: 22,
+        accommodationId: 142,
+        roomType: 'clasic room',
+        numberOfBeds: 1,
+        departureDate: '2020-04-04',
+        checkoutDate: '2020-04-12'
+      })
+      .end((err, res) => {
+        res.should.have.status(201);
```

Comment:
@VedaDom Since you know the data you sent, you can also check if the response returns the data you sent.